v0/destinations/shynet/data/ShynetPageConfig.json

```
@@ -0,0 +1,17 @@
+[
+  {
+    "destKey": "loadTime",
+    "sourceKeys": "context.page.loadTime",
```

We don't have `loadTime` in `context.page` 

```suggestion
    "sourceKeys": "properties.loadTime"
```